documentcloud/documents/rules/entities.py

```diff
@@ -1,7 +1,15 @@
+# pylint: disable=unused-argument, invalid-unary-operand-type
+
+
 # Third Party
 from rules import add_perm, always_allow, always_deny, is_authenticated
 
+# DocumentCloud
+from documentcloud.core.rules import skip_if_not_obj
+
 add_perm("documents.view_entityoccurrence", always_allow)
 add_perm("documents.add_entityoccurrence", is_authenticated)
 add_perm("documents.change_entityoccurrence", always_deny)
 add_perm("documents.delete_entityoccurrence", is_authenticated)
+add_perm("documents.view_entity", always_allow)
+add_perm("documents.add_entity", is_authenticated)
```

Comment:
This should be fine for now, but I do think we need to think more about how the permissions should work.